src/node.cc

```diff
@@ -1236,18 +1165,6 @@ void SetupDomainUse(const FunctionCallbackInfo<Value>& args) {
   HandleScope scope(env->isolate());
   Local<Object> process_object = env->process_object();
 
-  Local<String> tick_callback_function_key = env->tick_domain_cb_string();
-  Local<Function> tick_callback_function =
-      process_object->Get(tick_callback_function_key).As<Function>();
-
-  if (!tick_callback_function->IsFunction()) {
-    fprintf(stderr, "process._tickDomainCallback assigned to non-function\n");
-    ABORT();
-  }
-
-  process_object->Set(env->tick_callback_string(), tick_callback_function);
-  env->set_tick_callback_function(tick_callback_function);
-
   CHECK(args[0]->IsArray());
   env->set_domain_array(args[0].As<Array>());
```

Comment:
We should be able to remove this too, including all uses of `domain_array`. As I see it`set_domain_array` is only used to set the `domain` property on resource object in C++. We now do that using `async_hooks`.

Reply:
Ack! good catch!